records_mover/records/job/hints.py

```diff
@@ -24,6 +24,7 @@ def config_name(self) -> str:
          'delimiter, the escape character, or a line terminator). '
          'default: never quote fields.')
 
+# TODO: Can these list be unified with the ones in records.types?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker